tests/e2e/create/create_suite_test.go

```diff
@@ -20,16 +20,24 @@ func Test_Create(t *testing.T) {
 const (
 	testdataDir = "./testdata/moduleconfig/"
 
-	invalidConfigs        = testdataDir + "invalid/"
-	duplicateResources    = invalidConfigs + "duplicate-resources.yaml"
-	missingNameConfig     = invalidConfigs + "missing-name.yaml"
-	missingVersionConfig  = invalidConfigs + "missing-version.yaml"
-	missingManifestConfig = invalidConfigs + "missing-manifest.yaml"
-	nonHttpsResource      = invalidConfigs + "non-https-resource.yaml"
-	resourceWithoutLink   = invalidConfigs + "resource-without-link.yaml"
-	resourceWithoutName   = invalidConfigs + "resource-without-name.yaml"
-	manifestFileref       = invalidConfigs + "manifest-fileref.yaml"
-	defaultCRFileref      = invalidConfigs + "defaultcr-fileref.yaml"
+	invalidConfigs             = testdataDir + "invalid/"
+	duplicateIcons             = invalidConfigs + "duplicate-icons.yaml"
+	duplicateResources         = invalidConfigs + "duplicate-resources.yaml"
+	missingNameConfig          = invalidConfigs + "missing-name.yaml"
+	missingVersionConfig       = invalidConfigs + "missing-version.yaml"
+	missingManifestConfig      = invalidConfigs + "missing-manifest.yaml"
+	missingDocumentationConfig = invalidConfigs + "missing-documentation.yaml"
+	missingRepositoryConfig    = invalidConfigs + "missing-repository.yaml"
+	missingIconsConfig         = invalidConfigs + "missing-icons.yaml"
+	nonHttpsRepository         = invalidConfigs + "non-https-repository.yaml"
+	nonHttpsDocumentation      = invalidConfigs + "non-https-documentation.yaml"
+	nonHttpsResource           = invalidConfigs + "non-https-resource.yaml"
+	resourceWithoutLink        = invalidConfigs + "resource-without-link.yaml"
+	resourceWithoutName        = invalidConfigs + "resource-without-name.yaml"
+	iconsWithoutLink           = invalidConfigs + "icons-without-link.yaml"
+	iconsWithoutName           = invalidConfigs + "icons-without-name.yaml"
```

Comment:
not used in tests

Reply:
Thanks! Added an e2e testcase with this.